contrib/python/kalman.py

```diff
@@ -0,0 +1,81 @@
+#
+# Simple Implementation of the Kalman Filter for 1D data, without any dependencies
+# Originally written in JavaScript by Wouter Bulten
+# Now rewritten in Python
+#
+# 2017
+#
+# license GNU LESSER GENERAL PUBLIC LICENSE v3
+# author Sifan Ye
+#
+# see https://github.com/wouterbulten/kalmanjs
+#
+
+import math
+
+class KalmanFilter:
+    A = 1
+    B = 0
+    C = 1
+
+    R = 0
+    Q = 0
+
+    cov = float('nan')
+    x = float('nan')
+
+    # Constructor
```

Comment:
Please use valid docstrings at all functions. You can for example use the numpy standard (http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html).